packages/loaders/graphql-file/src/index.ts

```diff
@@ -31,6 +36,10 @@ function isGraphQLImportFile(rawSDL: string) {
   return trimmedRawSDL.startsWith('# import') || trimmedRawSDL.startsWith('#import');
 }
 
+function createGlobbyOptions(options: GraphQLFileLoaderOptions): GlobbyOptions {
```

Comment:
used by both code file and graphql file loaders, _might_ be a candidate for moving to the utils package? or the filesystem glob calls rather than something specific to globby
not sure and not advocating for the move 😅

Reply:
Sure 👍